config/loadConfig.ts

```diff
@@ -64,6 +64,10 @@ export function loadConfig(): Config {
               env.NEXT_PUBLIC_SENTRY_TRACES_SAMPLE_RATE
           ) ?? 0.2,
       },
+      stripeConfig: {
+        isEnabled:
+          parseBooleanEnvValue(env.NEXT_PUBLIC_STRIPE_ENABLED) ?? false,
```

Comment:
See the other configuration values for reference.

```suggestion
        isEnabled:
          parseBooleanEnvValue(env.STRIPE_ENABLED) ??
          parseBooleanEnvValue(env.NEXT_PUBLIC_STRIPE_ENABLED) ??
          false,
```